src/components/HeartbeatBar.vue

```diff
@@ -115,6 +146,49 @@ export default {
             };
         },
 
+        timeStyle() {
+            return {
+                "margin-left": this.numPadding * (this.beatWidth + this.beatMargin * 2) + "px",
+            };
+        },
+
+        /**
+         * Calculates the time elapsed since the first valid beat.
+         *
+         * @return {string} The time elapsed in minutes or hours.
+         */
+        firstBeatElapsed() {
```

Comment:
Not a strong opinion, but I would prefer `timeSinceFirstBeat` or `elapsedTimeSinceFirstBeat`, as this would be a bit more obvious what is meant just from the name